Source/Atomic/Graphics/Direct3D9/D3D9Graphics.cpp

```diff
@@ -941,6 +946,7 @@ void Graphics::DrawInstanced(PrimitiveType type, unsigned indexStart, unsigned i
     impl_->device_->DrawIndexedPrimitive(d3dPrimitiveType, baseVertexIndex, minVertex, vertexCount, indexStart, primitiveCount);
 
     numPrimitives_ += instanceCount * primitiveCount;
+    numSinglePassPrimitives_ += instanceCount * primitiveCount;
     ++numBatches_;

```

Comment:
Ok for D3D9, in order to use with D3D11/OpenGL the metrics gathering needs to be added to D3D11 and OpenGL systems